lib/goodcity/organisation_populator.rb

```diff
@@ -12,7 +12,6 @@ class OrganisationPopulator
 
     def initialize
       @organisation_type = get_organisation_type
-      @country           = Country.find_by_name_en(COUNTRY_NAME_EN)
```

Comment:
@trojanh In general, the initialize function shouldn't be used to do any 'work' e.g. database calls, http lookups, file loads etc. It should be just enough code to return a new class that hasn't done anything yet but is in a ready and consistent state to be called. E.g. imagine if we added another top level method to count the number of organisations that exist, we wouldn't need to find the country or load the URL just to do this so it would be extra unnecessary work when calling initialize.